src/org/labkey/snd/SNDManager.java

```diff
@@ -2685,7 +2687,13 @@ public Event updateEvent(Container c, User u, Event event, boolean validateOnly)
                                 UserSchema schema = getSndUserSchema(c, u);
                                 TableInfo eventTable = getTableInfo(schema, SNDSchema.EVENTS_TABLE_NAME);
                                 QueryUpdateService eventQus = getQueryUpdateService(eventTable);
-                                QueryUpdateService eventNotesQus = getNewQueryUpdateService(schema, SNDSchema.EVENTNOTES_TABLE_NAME);
+                                QueryUpdateService eventNotesQus = null;
+
+                                // make sure there is an event note to insert - event notes are optional - trim spaces
+                                if (event.getEventNotesRow(c).containsKey("note") && event.getEventNotesRow(c).get("note") != null && event.getEventNotesRow(c).get("note").toString().trim().length() > 0)
+                                    eventNotesQus = getNewQueryUpdateService(schema, SNDSchema.EVENTNOTES_TABLE_NAME);
+
+                                //QueryUpdateService eventNotesQus = getNewQueryUpdateService(schema, SNDSchema.EVENTNOTES_TABLE_NAME);
```

Comment:
We can probably just delete this commented line right?

Reply:
I'll delete it